src/language-js/printer-estree.js

```diff
@@ -1558,6 +1558,19 @@ function printPathNoParens(path, options, print, args) {
           n.elements.length > 1 &&
           n.elements.every((element, i, elements) => {
             const elementType = element && element.type;
+
+            if (n.type === "ArrayExpression") {
+              let i = 0;
+              let node = path.getNode(i);
+              while (node) {
+                const parent = path.getParentNode(i++);
+                if (parent && isJestEachTemplateLiteral(node, parent)) {
+                  return false;
+                }
+                node = parent;
```

Comment:
This loop back to the root, can we break the loop when meet an expression or something safe?

Reply:
@fisker by root, do you mean moving this logic outside `printPathNoParens()` function?